src/test/java/ru/training/at/hw2/ex1/HeaderAndMenuTextTest.java

```diff
@@ -0,0 +1,109 @@
+package ru.training.at.hw2.ex1;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.util.List;
+
+public class HeaderAndMenuTextTest {
+
+    @Test
+    public void headerAndMenuTextTest() {
+        System.setProperty("webdriver.chrome.driver", "./src/test/resources/chromedriver.exe");
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
```

Comment:
It's a code duplication. Think what can you move to common class(abstract?) And where are hooks?